docs/install_guides/installing-openvino-conda.md

```diff
@@ -1,10 +1,102 @@
-# Install OpenVINO™ Runtime from Anaconda Cloud
+# Install OpenVINO™ Runtime from Anaconda Cloud {#openvino_docs_install_guides_installing_openvino_conda}
 
-* [Install OpenVINO Runtime from an Archive File](installing-openvino-from-archive-linux.md)
-* [Install OpenVINO from PyPI](installing-openvino-pip.md)
-* [Install OpenVINO with Docker](installing-openvino-docker-linux.md)
-* [Build From Source](https://github.com/openvinotoolkit/openvino/blob/master/docs/dev/build.md)
+@sphinxdirective
 
-The other installation methods are temporarily unavailable.
+.. note:
+
+   Installing OpenVINO Runtime from Anaconda Cloud is recommended for C++ developers, as it provides only the C++ Runtime API.
+   If you work with Python, consider :doc:`installing OpenVINO from PyPI <openvino_docs_install_guides_installing_openvino_pip>`
+
+   The current Anaconda package does not provide support for GPU inference.
+
+.. tab:: System Requirements
+
+  | Full requirement listing is available in:
+  | `System Requirements Page <https://www.intel.com/content/www/us/en/developer/tools/openvino-toolkit/system-requirements.html>`__
+
+.. tab:: Processor Notes
+
+  | Not all Intel CPUs include integrated graphics processors.  
+  | See `Product Specifications <https://ark.intel.com/>`__ for information about your hardware.
+
+.. tab:: Software
+
+   There are many ways to work with Conda. Before you proceed, learn more about it on the
+   `Anaconda distribution page <https://www.anaconda.com/products/individual/>`__
+
+
+Installing OpenVINO Runtime with Anaconda Package Manager
+############################################################
+
+1.  Set up the Anaconda environment (Python 3.7 used as an example):
+   
+   .. code-block:: sh
+
+      conda create --name py37 python=3.7
+
+   .. code-block:: sh
+
+      conda activate py37
+
+2. Update it to the latest version:
+   
+   .. code-block:: sh
+
+      conda update --all
+
+3. Install the OpenVINO Runtime package:
+
+   .. code-block:: sh
+
+      conda install -c conda-forge openvino=2022.3.0
+
+4. Verify that the package is installed:
+   
+   .. code-block:: sh
+
+      is this a correct verification command?
+      
+      conda -c "from openvino.runtime import Core"
```

Comment:
please, remove this part. Python API is not provided as a part of 2022.3 / 2023.0 releases

Reply:
do we have a different command that would verify successful installation?